images/mta-operator.yml

```
@@ -1,32 +1,27 @@
-cachito:
-  packages:
-    gomod:
-      - path: .
 content:
   source:
     dockerfile: konflux.Dockerfile
     git:
       branch:
-        target: oadp-1.5
-      url: git@github.com:openshift-priv/oadp-operator.git
-      web: https://github.com/openshift/oadp-operator
+        target: mta-8.0
```

❗  **Across images/** 

```suggestion
        target: release-0.8
```
This is expected to follow existing MTA/Tackle release branch naming pattern; similar to MTC (`release-1.8`) rather than OADP (`oadp-1.5`).

---

Fixed in all components using release-0.8 which maps to mta-8.0